docs/ChannelValue.md

```diff
@@ -43,9 +43,40 @@ Any other props supplied will be provided to the root element (`span`).
 
 You can override the classes used by Brightlayer UI by passing a `classes` prop. It supports the following keys:
 
-| Name  | Description                         |
-| ----- | ----------------------------------- |
-| root  | Styles applied to the root element  |
-| icon  | Styles applied to the icon element  |
-| units | Styles applied to the units element |
-| value | Styles applied to the value element |
+| Name   | Description                          |
+| ------ | ------------------------------------ |
+| root   | Styles applied to the root element   |
+| icon   | Styles applied to the icon element   |
+| text   | Styles applied to the text element   |
+| prefix | Styles applied to the prefix element |
+| suffix | Styles applied to the suffix element |
+| units  | Styles applied to the units element  |
+| value  | Styles applied to the value element  |
+
+### `sx` Class Overrides
+
+You can override the styles used by Brightlayer UI by passing a `sx` prop. The `sx` prop styles will override styles provided through the `Classes` prop. It supports the following classNames:
+
+| Selector                 | Description                          |
+| ------------------------ | ------------------------------------ |
+| .BluiChannelValue-icon   | Styles applied to the icon element   |
+| .BluiChannelValue-text   | Styles applied to the text element   |
+| .BluiChannelValue-prefix | Styles applied to the prefix element |
+| .BluiChannelValue-suffix | Styles applied to the suffix element |
```

Comment:
What is a prefix element? What is a suffix element?

Reply:
I get what you're getting at. There is a reference to what prefix is in the API and image at the top, but how would you suggest I describe what a prefix and suffix are?

Reply:
Would something such as "Styles applied to the unit element when used as a prefix|suffix" work as a description

Reply:
Looking at MUI's naming convention, I think the class names might make more sense this way:

`.BluiChannelValue-prefixUnit` `.BluiChannelValue-prefixValue`
`.BluiChannelValue-suffixUnit` `.BluiChannelValue-suffixValue`
(Yeah that's a lot)
or 
`.BluiChannelValue-prefixText` `.BluiChannelValue-suffixText`

If you just mention `.BluiChannelValue-prefix` it usually means "styles applied to the root element if `prefix={true}`"

You can also introduce an alias of `prefix` like `prefixed` to make the English flow better

Sorry just dumping all my thoughts. Or maybe you can choose not to advertise about it. I don't know man, these two classes feel strange